Libplanet.Explorer/Queries/TransactionQuery.cs

```diff
@@ -107,6 +122,164 @@ public TransactionQuery()
                 }
             );
 
+            Field<NonNullGraphType<ByteStringType>>(
+                name: "unsignedTransaction",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "publicKey",
+                        Description = "The hexadecimal string of public key for Transaction.",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "plainValue",
+                        Description = "The hexadecimal string of plain value for Action.",
+                    },
+                    new QueryArgument<LongGraphType>
+                    {
+                        Name = "nonce",
+                        Description = "The nonce for Transaction.",
+                    }
+                ),
+                resolve: context =>
+                {
+                    BlockChain<T> chain = _context.BlockChain;
+                    string plainValueString = context.GetArgument<string>("plainValue");
+                    IValue plainValue = _codec.Decode(ByteUtil.ParseHex(plainValueString));
+                    var action = new T();
+                    action.LoadPlainValue(plainValue);
+
+                    var publicKey = new PublicKey(
+                        ByteUtil.ParseHex(context.GetArgument<string>("publicKey"))
+                    );
+                    Address signer = publicKey.ToAddress();
+                    long nonce = context.GetArgument<long?>("nonce") ??
+                        chain.GetNextTxNonce(signer);
+                    Transaction<T> unsignedTransaction =
+                        Transaction<T>.CreateUnsigned(
+                            nonce,
+                            publicKey,
+                            chain.Genesis.Hash,
+                            new[] { action }
+                        );
+                    return unsignedTransaction.Serialize(false);
+                }
+            );
+
+            Field<NonNullGraphType<StringGraphType>>(
+                name: "bindSignature",
+                #pragma warning disable MEN002
```

Comment:
How about suppressing it for the whole method?

Reply:
It may be a kind of choice between code verbosity and keeping warning areas...
In my point of view, the code verbosity of this sort can be bearable, but warning diabling isn't.

but on the other hands, it's totally personal opinion and increasing region isn't critical for even me. 😄 

@planetarium/libplanet any ideas?